.github/workflows/test-suite.yml

```
@@ -309,7 +309,7 @@ jobs:
       run: |
           make
     - name: Install lcli
-      # TODO: uncomment after the version of lcli in https://github.com/sigp/lighthouse/pull/5137 
+      # TODO: uncomment after the version of lcli in https://github.com/sigp/lighthouse/pull/5137
       # is installed on the runners
       # if: env.SELF_HOSTED_RUNNERS == 'false'
       run: make install-lcli
```

```suggestion
      if: env.SELF_HOSTED_RUNNERS == 'false'
      run: make install-lcli
```

ah maybe we can sneak this in

---

We actually need to keep this commented out for now, it suffers from the same problem as #5137 :upside_down_face:

---

oh right!